pkg/models/runtime_env_credential.go

```
@@ -0,0 +1,77 @@
+	return &RuntimeEnvCredential{
+		RuntimeEnvCredentialId: NewRuntimeEnvCrentialId(),
+		Name:        name,
+		Description: description,
+		Owner:       owner,
+		Content:     RuntimeEnvCredentialContentMapToString(content),
+		Status:      constants.StatusActive,
+		CreateTime:  time.Now(),
+		StatusTime:  time.Now(),
+	}
+}
+
+func RuntimeEnvCredentialToPb(runtimeEnvCredential *RuntimeEnvCredential) *pb.RuntimeEnvCredential {
+	pbRuntimeEnvCredential := pb.RuntimeEnvCredential{}
+	pbRuntimeEnvCredential.RuntimeEnvCredentialId = utils.ToProtoString(runtimeEnvCredential.RuntimeEnvCredentialId)
+	pbRuntimeEnvCredential.Name = utils.ToProtoString(runtimeEnvCredential.Name)
+	pbRuntimeEnvCredential.Description = utils.ToProtoString(runtimeEnvCredential.Description)
+	pbRuntimeEnvCredential.Owner = utils.ToProtoString(runtimeEnvCredential.Owner)
+	pbRuntimeEnvCredential.Status = utils.ToProtoString(runtimeEnvCredential.Status)
+	pbRuntimeEnvCredential.CreateTime = utils.ToProtoTimestamp(runtimeEnvCredential.CreateTime)
+	pbRuntimeEnvCredential.StatusTime = utils.ToProtoTimestamp(runtimeEnvCredential.StatusTime)
+	pbRuntimeEnvCredential.Content = RuntimeEnvCredentialContentStringToMap(runtimeEnvCredential.Content)
+	return &pbRuntimeEnvCredential
+}
+
+func RuntimeEnvCredentialToPbs(runtimeEnvCredentials []*RuntimeEnvCredential) (pbRuntimeEnvCredentials []*pb.RuntimeEnvCredential) {
+	for _, runtimeEnvCredential := range runtimeEnvCredentials {
+		pbRuntimeEnvCredentials = append(pbRuntimeEnvCredentials, RuntimeEnvCredentialToPb(runtimeEnvCredential))
+	}
+	return
+}
+
+func RuntimeEnvCredentialContentStringToMap(stringContent string) map[string]string {
+	var mapContent map[string]string
+	json.Unmarshal([]byte(stringContent), &mapContent)
+	return mapContent
+}
+
+func RuntimeEnvCredentialContentMapToString(mapContent map[string]string) string {
+	stringContent, _ := json.Marshal(mapContent)
```

pls do not ignore any error!
return err, or log error, or panic.